nemo/deploy/service/rest_model_api.py

```diff
@@ -66,6 +76,23 @@ class CompletionRequest(BaseModel):
     frequency_penalty: float = 1.0
 
 
+@app.get("/triton_health")
+async def check_triton_health():
+    """
```

Comment:
Added a test method `check_triton_health` for users to verify if Pytriton server is up and running since many times while using REST service, we encounter an error if Pytriton server is not accessible. This test method adds a check for that.